index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Youtube clone</title>
+    <link rel=“stylesheet”
+    href=“https://use.fontawesome.com/releases/v5.5.0/css/all.css”
+    integrity=“sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU”
+    crossorigin=“anonymous”>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+
+    <link href="favicon.icon" rel="icon" type="image/x-icon" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <!-- main -->
+
+    <!-- navbar -->
+    <div id="navbar">
+      <div class="bars">
+        <i class="fas fa-bars"></i>
+      </div>
+      <div id="logo">
+        <img src="youtube.svg" />
+        <h4>YouTube</h4>
+      </div>
+
+      <div class="search-bar">
+        <input type="text" placeholder="Search" />
+        <button>
+          <i class="fas fa-search"></i>
+        </button>
+      </div>
+      <div>
+        <ul>
+          <li><i class="fas fa-video"></i></li>
+          <li><i class="fas fa-th"></i></li>
+          <li><i class="fas fa-ellipsis-v"></i></li>
+        </ul>
+      </div>
+    </div>
+    <div class="main">
+      <!-- main container -->
+      <div class="main-container">
+        <!-- video section -->
+        <div class="video-section">
+          <div class="video">
+            <iframe
+              width="900"
+              height="506"
+              src="https://www.youtube.com/embed/NZ2_mOTc4TM"
+              frameborder="0"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </div>
+          <div class="embedded-video"></div>
+          <div class="video-title">
+            Milan 4-2 Juventus | Ronaldo Goal Not Enough as Milan Stun Serie A
+            Leaders! | Serie A TIM
+          </div>
+          <div class="video-info">
+            <div class="vid-info-start">
+              11,768,672 views • Jul 8, 2020
+            </div>
+
+            <ul class="vid-info-end">
+              <li><i class="fas fa-thumbs-up"></i> 211K</li>
+              <li><i class="fas fa-thumbs-down"></i> 8K</li>
+              <li><i class="fas fa-share"></i> SHARE</li>
```

Comment:
The share should be a button
- [ ] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))